internal/broker/broker.go

```diff
@@ -110,11 +110,13 @@ func New(cfg Config, args ...Option) (b *Broker, err error) {
 	}
 
 	if cfg.CachePath == "" {
-		return &Broker{}, errors.New("cache path must be provided")
+		return &Broker{}, errors.New("cache path is required and was not provided")
 	}
-
-	if cfg.IssuerURL == "" || cfg.ClientID == "" {
-		return &Broker{}, errors.New("issuer and client ID must be provided")
+	if cfg.IssuerURL == "" {
+		return &Broker{}, errors.New("issuer URL is required and was not provided")
+	}
+	if cfg.ClientID == "" {
+		return &Broker{}, errors.New("client ID is required and was not provided")
```

Comment:
maybe we should rather check both and then join the errors? It’s annoying when you get an error, fix it, restart, and get another error :p